### DISCUSSION TITLE
Sponsorship request stuck in "Pending" state

### DISCUSSION THREAD

**Discussion by alirezazolanvari** [community/community#134152] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I'm reaching out for assistance with my GitHub Sponsors account approval process. I submitted my request a few months ago, but it's been stuck in the "Pending" status since then. The only information that I see in my dashboard is:

> You've submitted your profile for approval Your profile will go live as soon as it's approved by a staff member.

Could someone from the GitHub Sponsors team please look into this and provide an update on the status of my application? Are there any additional steps I can take to speed up the approval process?

**Comment by alamenai** [comment]
Same for me !

**Comment by Harries** [comment]
Same as me，How long it takes to review is a mystery

**Comment by alamenai** [comment]
For my side, I guess they are rejected me due my current location though the provided information are acceptable, but the state remains pending , I guess it's a bug from GitHub to init the state.

**Comment by nickf2k** [comment]
same to me.
Does anyone know how long it will be reviewed by Github team?

**Comment by alirezazolanvari** [comment, marked as answer]
Typically, it shouldn't take more than 2 weeks. In my case, I reached out to GitHub support directly and they identified an issue with my fiscal host. If your request remains in a pending status for over 2 weeks, I recommend contacting GitHub support.

**Reply by alamenai** [reply]
Yes, in my case, they told me to use fiscal host instead of GitHub Sponsorship. They are using Stripe while stripe already added new supported countries but Github is not sync with them - I guess.
